packages/core/admin/admin/src/translations/en.json

```
@@ -689,7 +689,9 @@
   "content-manager.form.Input.search.field": "Enable search on this field",
   "content-manager.form.Input.sort.field": "Enable sort on this field",
   "content-manager.form.Input.sort.order": "Default sort order",
-  "content-manager.form.Input.wysiwyg": "Display as WYSIWYG",
+  "content-manager.form.Input.hint.text": "{min, select, undefined {} other {min. {min}}}{hasMinAndMax, select, true { {divider} } other {}}{max, select, undefined {} other {max. {max}}}{hasMinOrMax, select, true { {unit}{br}} other {}}{description}",
+  "content-manager.form.Input.hint.minMaxDivider": "/",
+  "content-manager.form.Input.hint.units": "{maxValue, plural, one {character} other {characters}}",
```

Small thing, but what do you think about changing the id to e.g. 
```suggestion
  "content-manager.form.Input.hint.character.unit": "{maxValue, plural, one {character} other {characters}}",
```

To make sure this can easily be differentiated from e.g. a number?